app/src/main/java/org/jellyfin/androidtv/util/KeyProcessor.java

```diff
@@ -291,17 +293,17 @@ public static PopupMenu createItemMenu(BaseRowItem rowItem, UserItemDataDto user
 
         //Not sure I like this but I either duplicate processing with in-line events or do this and
         // use a single event handler
-        mCurrentItem = item;
+        mCurrentItem = rowItem.getBaseItem();
         mCurrentRowItemNdx = rowItem.getIndex();
-        mCurrentItemId = item.getId();
+        mCurrentItemId = item.getId().toString();
```

Comment:
Why remove item from L294 but keep it here?

Reply:
`rowItem.getBaseItem()` is the old BaseItemDto (from the apiclient) which we're migrating from. So I use the SDK one when I can, in this case I can't use it for mCurrentItem but can for the id.